kubeconfig.go

```diff
@@ -62,6 +65,61 @@ func (k *KubeConfig) NewRESTConfigForApp(ctx context.Context, app v1alpha1.App)
 	return restConfig, nil
 }
 
+func (k *KubeConfig) NewRESTConfigForKubeConfig(ctx context.Context, kubeConfig []byte) (*rest.Config, error) {
+	restConfig, err := clientcmd.RESTConfigFromKubeConfig(kubeConfig)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+	return restConfig, nil
+}
```

Comment:
Nitpicky but the methods should to be ordered. So `NewKubeConfigForRESTConfig` should be first.